cmd/lib.go

```diff
@@ -104,10 +108,8 @@ func getStoreCommon(hst host.Host) storePkg.Store {
 func init() {
 	resetFlagsFns = append(resetFlagsFns, func() {
 		ssh = defaultSsh
-		dockerContainer = defaultDockerContainer
-		dockerUser = defaultDockerUser
-		sudo = defaultSudo
-		disableAgent = defaultDisableAgent
+		docker = defaultDocker
+		options = defaultOptions
```

Comment:
worth a word on why this fuc@!$!@ is here: Cobra is great, but when testing, it does NOT reset these... and one test can taint the results of another... in the absence of a better solution, this is what we have.

Reply:
I don't quite follow what you mean here :stuck_out_tongue:

Reply:
I just wanted to explain why this code exist, it does look odd even to me who wrote it, lol.